Makefile

```diff
@@ -359,5 +360,18 @@ clean-gui:
 	- rm -rf jujugui/static/gui/build
 
 .PHONY: clean-all
-clean-all: clean-venv clean-pyc clean-gui clean-dist
+clean-all: clean-venv clean-pyc clean-gui clean-dist clean-uitest
 	- rm -rf *.egg-info
+
+.PHONY: clean-uitest
+clean-uitest:
+	- rm -rf uitest
+
+.PHONY: uitest
+uitest: sudo dist
+	@scripts/uitest.sh $(ARGS)
+
+.PHONY: sudo
+sudo:
+	@# Pre-cache sudo permissions, so that the process is not stopped later.
+	@sudo true
```

Comment:
`sudo -v` could work here to pre-cache permissions and update their timestamp while also not running a command.